ai-services/cmd/ai-services/cmd/application/delete.go

```diff
@@ -57,44 +86,52 @@ func deleteApplication(client *podman.PodmanClient, appName string) error {
 		pods = val
 	}
 
-	if len(pods) == 0 {
-		logger.Infof("No pods found with given application: %s\n", appName)
-		return nil
-	}
+	podsExists := len(pods) != 0
 
-	logger.Infof("Found %d pods for given applicationName: %s.\n", len(pods), appName)
-	logger.Infoln("Below are the list of pods to be deleted")
-	for _, pod := range pods {
-		logger.Infof("\t-> %s\n", pod.Name)
-	}
+	if podsExists {
+		logger.Infof("Found %d pods for given applicationName: %s.\n", len(pods), appName)
+		logger.Infoln("Below are the list of pods to be deleted")
+		for _, pod := range pods {
+			logger.Infof("\t-> %s\n", pod.Name)
+		}
 
-	confirmDelete, err := utils.ConfirmAction("Are you sure you want to delete above pods? ")
-	if err != nil {
-		return fmt.Errorf("failed to take user input: %w", err)
-	}
+		confirmDelete, err := utils.ConfirmAction("Are you sure you want to delete above pods? ")
```

Comment:
I see the confirmation is only for the deletion of the pod, how you want to handle the case?

option 1: generic confirmation message for an entire flow including the message saying you are deleting the pod + data

option 2: confirmation for pod delete + confirmation for data

Reply:
done, ptal